#### Expression Quantitative Trait Loci

The use of genomic improvement techniques has advanced pork production significantly. Quantitative genetics has revealed genomic regions harboring single nucleotide polymorphisms (SNPs) contributing significant portions of phenotypic variation. However, the molecular mechanisms underlying these contributions remain unclear. We aim to identify expression quantitative trait loci (eQTL) in the F2 generation of the Michigan State University Pig Resource Population (MSUPRP) to elucidate the genetic architecture of gene expression variation and gain insight on the genetic contribution underlying differences in polygenic traits. Associating gene expression profiles to genome-wide SNP markers reveals both local and distal regulators of gene expression. Local eQTL identify candidate loci directly influencing the expression of the associated gene and thus infers direct cause of variation in gene expression. Distal eQTL identify candidate gene-gene interactions where the expression of one gene may affect the expression of distant genes through transcriptional co-regulation. As sequencing costs go down, eQTL studies have the potential to be incorporated into expression-assisted evaluations in animal breeding by the use of genetical genomic selection of both phenotypic QTL and expression QTL.

Our lab is currently conducting eQTL studies utilizing genotype and transcriptomic data (mRNA and miRNA) from adipose and skeletal muscle tissues to identify genomic loci influencing gene expression and economically-important pig-production phenotypes related to growth, meat quality, and carcass merit. 


#### RNA Editing

Among the complex ways in which animals can regulate the expression of transcripts, endogenous RNA editing has been shown to be widespread, catalyzed primarily by the nuclear expressed family of adenosine deaminase acting on RNA (ADAR) proteins that perform single adenosine to inosine transitions along premature RNA transcripts. It remains unknown to what degree transcriptome-wide RNA editing exists in livestock species and its overall effect on controlling complex traits. We have built bioinformatic software to streamline the RNA editing detection process from whole genome sequencing and RNA-seq data and have begun to apply it toward the pig model, observing inherent associations between candidate RNA editing sites and swine-specific SINE retrotransposons. This work is crucial to understanding the extent to which RNA editing contributes to single nucleotide variation among the swine transcriptome, and ongoing work of ours to analyze the “editomes” of various species can reveal new insights into how distinct retrotransposon sequences influence RNA editing patterns.